corelib/dynamicemb/example/run_example.sh

```diff
@@ -1,10 +1,5 @@
 #!/bin/bash
 
-export RANK=0
-export WORLD_SIZE=1
-export MASTER_ADDR=127.0.0.1
-export MASTER_PORT=29500
-export LOCAL_WORLD_SIZE=1
-
-python example.py --use_embedding_collection
-python example.py --use_embedding_bag_collection
\ No newline at end of file
+torchrun --nproc_per_node=4 example.py --train 
```

Comment:
how  about 
torchrun --standalone --nproc_per_node=${NGPU} example.py "$@"

then you can call with different args without modify the sh, 
NGPU=4 ./run_example.sh --epochs 10 --lr 3e-4